src/encode_task_pool_ta.py

```diff
@@ -53,7 +53,7 @@ def pool_ta(tas, col, basename_prefix, out_dir):
         run_shell_cmd(cmd)
         return pooled_ta
     else:
-        return make_hard_link(tas[0], out_dir)
+        raise Exception('Needs at least two TAs (or BEDs) to be pooled.')
```

Comment:
This should probably be a ValueError:
```suggestion
        raise ValueError('Needs at least two TAs (or BEDs) to be pooled.')
```